### ISSUE TITLE
Blinx 2 - Masters of Time & Space - Ocean water in first stage is the wrong color

### ISSUE THREAD

**Issue by ehw** [xemu-project/xemu#1405]
### Title

https://xemu.app/titles/4d530065/#Blinx-2-Battle-of-Time-Space-Blinx-2-Masters-of-Time-Space
Blinx 2 - Masters of Time & Space ~ Blinx 2 - Battle of Time & Space (World) (En,Ja,Fr,De,Es,It)
MS-101 [4d530065]

### Bug Description

The water color on the ocean in the first stage is black. 

You can easily see the water texture by doing the following:
1.) Start the game up and idle at the title screen for a few seconds.
2.) The game will then load a small demo of the first stage where you can see the water.

### Expected Behavior

The texture should be clear blue.

https://youtu.be/DYrKm85otjA?t=1590

### xemu Version

Version:      0.7.85
Branch:       master
Commit:       065c74a00fe402f65147e9b0d1a1eeac1b4b609f
Date:         Mon Mar  6 21:06:28 UTC 2023

### System Information

CPU:          AMD Ryzen 7 5800X 8-Core Processor             
OS Platform:  Windows
OS Version:   Windows
Manufacturer: NVIDIA Corporation
GPU Model:    NVIDIA GeForce RTX 3080/PCIe/SSE2
Driver:       4.0.0 NVIDIA 531.26
Shader:       4.00 NVIDIA via Cg compiler

### Additional Context

![image](https://user-images.githubusercontent.com/885618/224511934-cd7cbb84-3cdd-43fe-9991-b471e1181c86.png)

**Comment by ehw** [comment]
Following up since it's been two years. The same issue is still occurring. It actually looks like the whole water effect is completely missing. The issue is identical between backends.

![Image](https://github.com/user-attachments/assets/81f954fa-8b61-4082-b2c2-fcf027017279)

**Comment by Triticum0** [comment]
@ehw seem to be fixed on latest build. Note needs #587 to be fully fixed. Suspected pr that fixed it #2129

![Image](https://github.com/user-attachments/assets/9cdcad2f-9d57-4ee3-b975-5c5d13fc92b3)

**Comment by ehw** [comment]
Yep! This is fixed. Although I'm not sure whether to close it or not if NV_PGRAPH_TEXFILTER_SIGNED needs to be fully fixed first.

**Comment by Triticum0** [comment]
I added blinks 2 to NV_PGRAPH_TEXFILTER_SIGNED issue. so close this as the issue been fixed.
